--- DistrhoUI.cpp	2021-06-17 22:55:07.000000000 +0200
+++ DistrhoUI_mac.cpp	2021-06-17 22:53:15.000000000 +0200
@@ -183,13 +183,13 @@
 
 /* ------------------------------------------------------------------------------------------------------------
  * UI Resize Handling, internal */
-
+extern float FIXME_MacScaleFactor();
 void UI::onResize(const ResizeEvent& ev)
 {
     UIWidget::onResize(ev);
 
-    const uint width = ev.size.getWidth();
-    const uint height = ev.size.getHeight();
+    const uint width = ev.size.getWidth() / FIXME_MacScaleFactor();
+    const uint height = ev.size.getHeight() / FIXME_MacScaleFactor();
     uiData->setSizeCallback(width, height);
 }
 #endif // !DISTRHO_PLUGIN_HAS_EXTERNAL_UI
